chaoskube/chaoskube.go

```diff
@@ -156,6 +159,14 @@ func (c *Chaoskube) Candidates() ([]v1.Pod, error) {
 		return nil, err
 	}
 
+	if c.MinimumAge > time.Duration(0) {
+		creationTime := c.Now().Add(-c.MinimumAge)
+		pods, err = filterByCreationTime(pods, creationTime)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Let's keep this function high-level and push most of into the `filterBy...` functions.

How about changing this to:

```go
pods = filterByMinimumAge(pods, c.MinimumAge) // you can remove the `err`, I accidentally introduced it.
```

and then in `filterByMinimumAge`

```go
func filterByMinimumAge(pods []v1.Pod, age time.Time) []v1.Pod {
 if age == 0 {
    return pods // early return unfiltered list
  }

  for _, pod := range pods {
  // ... like you already have
```